builder/builder-module.c

```diff
@@ -895,6 +896,8 @@ builder_module_download_sources (BuilderModule  *self,
       if (!builder_source_is_enabled (source, context))
         continue;
 
+      builder_set_term_title ("%s %s", _("Downloading"), self->name);
```

Comment:
Isn’t the fact that `"%s %s"` is not translatable going to be an issue in RtL locales?